src/cli.ts

```diff
@@ -125,6 +121,7 @@ export async function downloadProfile(
     logger.info(`File to be pulled: ${file}`);
 
     // If destination path is not specified, pull to the current directory
+    // @ts-ignore
```

Comment:
what typescript error do we ignore here? In general it's better to throw an error, than to ignore an error (that being said we might want to move the `packageNameWithSuffix` code down into the if-block where it's actually needed.

Because lets assume a file is provided, then we don't need to fail, just because we couldn't resolve the package name.

Reply:
> what typescript error do we ignore here?

Actually none 😅 because I didn't type return value from `getConfig()` function (it's `any`), I don't know if you want to move here the whole `Config` type. I think that for your use case just checking `packageNameWithSuffix` would work 👍